whelk-core/src/main/groovy/whelk/component/ElasticSearch.groovy

```diff
@@ -600,7 +605,7 @@ class ElasticSearch {
 
         @Override
         String queryPath() {
-            "/${indexName}/_search?filter_path=$filterPath"
+            "/${indexName}/_search?search_type=$SEARCH_TYPE&filter_path=$filterPath"
```

Comment:
Can this just call `getQueryUrl` now? ("Eliminate moving parts" or are they "accidentally equal" right now?)

Reply:
Let's say they're unaccidentally equal